go.mod

```diff
@@ -24,49 +24,53 @@ require (
 	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20200321030439-57b580e57e88 // 0.11.0
 	github.com/operator-framework/operator-sdk v0.17.0
 	github.com/rook/rook v1.4.6
-	github.com/sirupsen/logrus v1.5.0
+	github.com/sirupsen/logrus v1.6.0
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
 	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
 	golang.org/x/lint v0.0.0-20200302205851-738671d3881b // indirect
 	golang.org/x/tools v0.0.0-20200915031644-64986481280e // indirect
-	google.golang.org/api v0.14.0
-	k8s.io/api v0.17.4
-	k8s.io/apiextensions-apiserver v0.17.4
-	k8s.io/apimachinery v0.17.4
-	k8s.io/cli-runtime v0.17.4
+	google.golang.org/api v0.15.1
+	k8s.io/api v0.19.3
+	k8s.io/apiextensions-apiserver v0.18.6
+	k8s.io/apimachinery v0.19.3
+	k8s.io/cli-runtime v0.19.3
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/code-generator v0.17.4
-	k8s.io/gengo v0.0.0-20191010091904-7fa3014cb28f
-	k8s.io/kube-openapi v0.0.0-20200204173128-addea2498afe
-	k8s.io/kubectl v0.17.4
+	k8s.io/code-generator v0.19.3
+	k8s.io/gengo v0.0.0-20200428234225-8167cfdcfc14
+	k8s.io/kube-openapi v0.0.0-20200805222855-6aeccd4b50c6
+	k8s.io/kubectl v0.18.2
 	nhooyr.io/websocket v1.7.4
-	sigs.k8s.io/controller-runtime v0.5.2
+	sigs.k8s.io/controller-runtime v0.6.3
 	sigs.k8s.io/yaml v1.2.0
 )
 
 // Pinned to kubernetes-1.17.4
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+incompatible // Required by OLM
```

Comment:
Do we still need this?

Reply:
seems like we do. didn't work on removing it now